pkg/devicemapper/devmapper.go

```diff
@@ -284,9 +284,9 @@ func FindLoopDeviceFor(file *os.File) *os.File {
 	return nil
 }
 
-func UdevWait(cookie uint) error {
-	if res := DmUdevWait(cookie); res != 1 {
-		logrus.Debugf("Failed to wait on udev cookie %d", cookie)
+func UdevWait(cookie *uint) error {
```

Comment:
It looks like this method is being called in a defer and the value for cookie is not yet set when the defer is initialized